licenses/templatetags/lang_pad_horizontal.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def lang_pad_horizontal(is_rtl, is_reverse):
+    """Adjust padding left or right based on language direction
+
+    Parameter(s):
+        - is_rtl - (boolean) whether or not the langauge of the
+                 - of the template is a right to left language
+    """
```

Comment:
Can you add an explanation of what `is_reverse` is for and when I should pass False or True?

Or maybe we could have two tags. (I may be remembering this from another project a few years ago.) Something like

    @register.simple_tag(takes_context=True)
    def start(context):
        return "right" if context["is_rtl"] else "left"

and another one `end` that returns the opposite.  Usage might look like `class="padding-{% start %}-normal"` if you wanted normal padding on the left normally and on the right if we're in RTL mode.

Reply:
done

Reply:
Originally, you asked me to include:
```
{% get_language_info for 'en' as lang %}
{% with is_rtl=lang.bidi %}
```
You said we didn't want the view to handle language direction. Just for it to supply the language. The above example is using context. Should the above code be removed as well?